src/library.js

```diff
@@ -2967,7 +2967,7 @@ LibraryManager.library = {
   // sys/mman.h
   // ==========================================================================
 
-  mmap__deps: ['$FS', 'memset'],
+  mmap__deps: ['$FS', 'memset', 'mmap_read', 'mmap_write'],
```

Comment:
If someone does mmap, always include read/write methods. This way we don't have to add these functions as dependencies on all the different read/write methods.